spec/requests/api/v1/moves_controller_destroy_spec.rb

```diff
@@ -3,6 +3,7 @@
 require 'rails_helper'
 
 RSpec.describe Api::V1::MovesController, with_client_authentication: true do
+  let!(:application) { Doorkeeper::Application.create(name: 'test') }
```

Comment:
This line looks like it's repeated quite a bit. Would it be a good candidate to refactor into (or in the same way as) `with_client_authentication`

Reply:
Solved.